tests/org.eclipse.jface.text.tests/META-INF/MANIFEST.MF

```diff
@@ -22,7 +23,10 @@ Require-Bundle:
  org.eclipse.text.tests;bundle-version="[3.5.0,4.0.0)",
  org.eclipse.core.runtime;bundle-version="[3.29.0,4.0.0)",
  org.eclipse.ui.workbench.texteditor,
- org.eclipse.test.performance;bundle-version="3.13.0"
+ org.eclipse.test.performance;bundle-version="3.13.0",
+ org.mockito.mockito-core;bundle-version="5.11.0",
+ net.bytebuddy.byte-buddy;bundle-version="1.14.13",
+ net.bytebuddy.byte-buddy-agent;bundle-version="1.14.13"
```

Comment:
Do you really need these?

Reply:
Yes. byte-buddy is needed for Mockito, Mockito is needed to mock the static class GC.

Reply:
Hm, I thought it should be automatically added since I didn't see any other references to byte-buddy in the WS (but several references to Mockito).
The tests are failing (invalid access). Can you please look into that?

Reply:
This is odd: no other bundle explicitly imports the bytebuddy bundle (only the package) and yet they all work. Also, by looking at the dependencies of the bundle `org.mockito.mockito-core` I see:

For version `5.11.0`:
![image](https://github.com/eclipse-platform/eclipse.platform.ui/assets/2205684/f54be07a-a487-4b43-a855-a7ba09523854)

For version `5.12.0`:
![image](https://github.com/eclipse-platform/eclipse.platform.ui/assets/2205684/5d330d3c-d9a8-41f3-a383-2d46d6bcdbe5)

Maybe the tests run locally with `5.11.0` and that extra dependency to `5.12.0` breaks stuff? 

Maybe you could investigate that further and try what happens if:
1. You explicitly set the version to `5.12.0`
2. You don't set a version (which one is used then?)
3. You set the minimum version to `2.13.0`, like other test bundles do (which one is used then?)

Reply:
I changed to 5.12 and removed the dependency to byte-buddy. On my local machine the unit tests doesn't work, but maybe it's caused by a odd local setup. Let's see if the build server can run the tests...

Reply:
Seems to work, at least the linux build does work. Windows and Mac should be unrelated. Thanks for your help 👍